### ISSUE TITLE
Event listeners get removed when re-opening qtip

### ISSUE THREAD

**Issue by johnnycrab** [qTip2/qTip2#543]
qTip's `_update` method empties the `element` which means that all events within the tooltip get removed. This is inconvenient when re-using the content (for example an interface within a tooltip)

See `PROTOTYPE._update`:

``` javascript
// Append new content if its a DOM array and show it if hidden
if(content.jquery && content.length > 0) {
    element.empty().append( content.css({ display: 'block' }) );
}
```

A fix for this would be substituting `empty()` with `children().detach()`. 
This would look like:

``` javascript
// Append new content if its a DOM array and show it if hidden
if(content.jquery && content.length > 0) {
    element.children()
        .detach()
        .end()
    .append( content.css({ display: 'block' }) );
}
```
